tests/BaseObjectTest.php

```diff
@@ -7,14 +7,12 @@
 
 use Friendica\App;
 use Friendica\BaseObject;
-// backward compatibility
```

Comment:
Are you sure that this works? It took me a while to get travis working with the three PHP versions.

Reply:
Ah ok, you the file below is loaded before... stupid github interface ;-)